src/pages/ThorChainLP/components/AddLiquitity/AddLiquityStatus.tsx

```diff
@@ -58,163 +37,69 @@ export const AddLiquidityStatus = ({ confirmedQuote }: AddLiquidityStatusProps)
   const assets: Asset[] = useMemo(() => {
     if (!(foundPool && asset && rune)) return []
 
-    if (foundPool.asymSide === null) return [asset, rune]
-    if (foundPool.asymSide === AsymSide.Rune) return [rune]
-    if (foundPool.asymSide === AsymSide.Asset) return [asset]
-
-    throw new Error('Invalid asym side')
+    switch (foundPool.asymSide) {
+      case null:
+        return [rune, asset]
+      case AsymSide.Rune:
+        return [rune]
+      case AsymSide.Asset:
+        return [asset]
+      default:
+        assertUnreachable(foundPool.asymSide)
+    }
   }, [asset, foundPool, rune])
 
   const handleGoBack = useCallback(() => {
     history.push(AddLiquidityRoutePaths.Input)
   }, [history])
 
   const handleComplete = useCallback(() => {
-    setSecondTx(TxStatus.Confirmed)
-    setIsComplete(true)
-  }, [])
-
-  const handleNext = useCallback(() => {
-    setFirstTx(TxStatus.Confirmed)
-    setSecondTx(TxStatus.Unknown)
-  }, [])
-
-  const hStackDivider = useMemo(() => {
-    if (foundPool?.asymSide) return <></>
-
-    return <RawText mx={1}>{translate('common.and')}</RawText>
-  }, [foundPool?.asymSide, translate])
-
-  const renderBody = useMemo(() => {
-    if (!(foundPool && asset && rune)) return null
-
-    if (isComplete) {
-      return (
-        <CardBody display='flex' flexDir='column' alignItems='center' justifyContent='center'>
-          <Center
-            bg='background.success'
-            boxSize='80px'
-            borderRadius='full'
-            color='text.success'
-            fontSize='xl'
-            my={8}
-          >
-            <FaCheck />
-          </Center>
-          <Heading as='h4'>{translate('pools.transactionSubmitted')}</Heading>
-        </CardBody>
-      )
-    }
+    setActiveStep(activeStep + 1)
```

Comment:
Can we add a comment here explaining the leetcody hack where `if (activeStep === assets.length)` then we are complete, as opposed to having a separate complete state field?